README.md

```diff
@@ -88,9 +88,10 @@ var options = {
 };
 ```
 
-#### `uncaughtException` (*Function*, optional)
+#### `uncaughtException` (*Function*, optional) **DEPRECATED**
 Handler for `uncaughtException` errors. See the [endgame](https://github.com/totherik/endgame) module for defaults.
 
+**Deprecated**: See `shutdown` middleware instead.
```

Comment:
Should we add more detail here or really just have people go look at the middleware module? If the latter, then maybe a link to it would be nice.